### ISSUE TITLE
OpenID login is broken

### ISSUE THREAD

**Issue by aaronpk** [osbridge/open-source-bridge-proposals#27]
Entering a URL that supports OpenID eventually results in a 504 Gateway Time-out error.

Entering a URL that does not have OpenID tags ends up with a "too many redirects" error, with a URL of `http://opensourcebridge.org/auth/failure?message=invalid_credentials&strategy=persona`

**Comment by aaronpk** [comment]
upon further investigation, it might be just my website that the server is having trouble contacting. Entering "http://tantek.com" as an OpenID identifier works and properly redirects to the OpenID server. However it gets hung up on "https://aaronparecki.com".

I'm still confused about the `persona` error though.

**Comment by znmeb** [comment]
1. Mozilla shut down Persona a while back.
2. I tried OpenID and couldn't log in with it either.

TL;DR: It's not you, it's them ;-)

I logged in with Twitter.

**Comment by aaronpk** [comment]
I don't know why I saw a persona error though. I didn't enter an email address, and was only using the OpenID login.

**Comment by znmeb** [comment]
Maybe it was defaulting to Persona?

**Comment by reidab** [comment]
There's definitely something going on there — I thought I'd removed the entire persona implementation from the auth flow, so I'm surprised that it would show up as a strategy. I'll take a look.
